samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -353,6 +356,8 @@ void main(void)
 		LOG_ERR("Setup failed, stopping.");
 		return;
 	}
+	(void)nrf_cloud_rest_alert_send(&rest_ctx, device_id, false,
```

Comment:
If I disable it before the message is sent, where would I do that?

If instead I call `nrf_cloud_rest_disconnect` after calling `nrf_cloud_rest_alert_send`, what does it gain me to set keep_alive right before?

Reply:
set `rest_ctx.keep_alive = false;` before the call to `send_message`.  it will disconnect automatically after the send.

if you enable the keep alive before `nrf_cloud_rest_alert_send`, then it will use the same connection for both `nrf_cloud_rest_alert_send` and `send_message`.

then call `nrf_cloud_rest_disconnect` after `send_message` if you don't want to use the keep_alive flag.